io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/reader/XLSInstanceReader.java

```diff
@@ -107,27 +105,74 @@ protected IOReport execute(ProgressIndicator progress, IOReporter reporter)
 		propAr = type.getChildren().toArray(new PropertyDefinition[type.getChildren().size()]);
 		Collection<List<String>> rows = analyser.getRows();
 
-		// skip if first row is a header
-		if (skip_N_lines == 0) {
-			// otherwise first line is also an instance
+		int skip_N_lines = 0;
+		Boolean skipType = getParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES).as(boolean.class);
+
+		if (skipType == null) {
```

Comment:
As in this section you can already handle all different cases, I think it would be cleaner to handle it similar to in the CSV reader, to translate the boolean values into numbers (0/1), and not treat them as different cases again further below.